traffic_ops/app/db/migrations/2021110207242921_admin_all_permission.up.sql

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This file and the corresponding `down` migration file should be renamed so that the timestamp is after the latest refetch migration.